src/Text/Pandoc/Readers/JATS.hs

```diff
@@ -368,9 +372,13 @@ parseBlock (Elem e) = do
          parseElement = filterChildren isEntry
          wrapWithHeader n mBlocks = do
                       isBook <- gets jatsBook
-                      let n' = if isBook || n == 0 then n + 1 else n
+                      case (filterChild (named “title”) e >>= attrValue “display-as”) of
+	                      Just l -> let n’ = decimal l
+	                      Nothing -> let n' = if isBook || n == 0 then n + 1 else n
                       headerText <- case filterChild (named "title") e of
-                                        Just t  -> getInlines t
+                                        Just t  -> case maybeAttrValue "supress" of 
+                                          Just s -> if s == "no" then getInlines t else return mempty 
+                                          Nothing -> getInlines t
```

Comment:
What is the `suppress` attribute supposed to do?

Reply:
@supress is a book-specific optional attribute of <title> that indicates wether the title should be displayed.